SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,119 @@
+# Map travel
+
+Before reading this topic, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).
+
+> Note: This topic is intended for advanced users only.
+
+## `APlayerController::ClientTravel`
+
+### In native Unreal
+`ClientTravel` is the process of changing which [map (or Level - see Unreal documentation)](http://api.unrealengine.com/INT/Shared/Glossary/index.html#l) a client currently has loaded.
+
+### User guide
```

Comment:
Can we say "In the GDK" as it is the other side of the "In native Unreal"?

Reply:
Sure